client/src/components/common/boost/BoostConfigurator.tsx

```diff
@@ -247,6 +258,24 @@ export default function BoostConfigurator({
               </Card>
             ))}
           </Accordion>
+          <Accordion className="small p-0 my-4">
+            <Card>
+              <Accordion.Toggle
```

Comment:
I reckon it'd be good to disable this toggle (hopefully that's something built into bootstrap) when there's no results - cleaner than having to open it up to reveal that there's nothing inside

Reply:
My only concern was that if it's not visible initially, then once generated people may not know to scroll down to view this toggle. If it's there from the start they know that the result are stored inside this button. How about disabling the button so it's not clickable when there are no results?

Reply:
Sorry I wasn't clear - that's exactly what I was thinking

Reply:
I found out that disabling is not a feature in Accordion, we would have to workaround it with our own code built on top. I think that would increase the complexity for not much benefit, based off some Stackoverflow posts. So for now I've just made it appear only when results are there. Let me know if that's okay.

Reply:
Should be fine!